docs/en/Tiered-Storage-on-Alluxio.md

```diff
@@ -106,55 +135,50 @@ existing allocators in Alluxio:
 In the future, additional allocators will be available. Since Alluxio supports custom allocators,
 you can also develop your own allocator appropriate for your workload.
 
-### Evictors
-
-Alluxio uses evictors for deciding which blocks to move to a lower tier, when space needs to be
-freed. Alluxio supports custom evictors, and implementations include:
-
-* **GreedyEvictor**
-
-    Evicts arbitrary blocks until the required size is freed.
-
-* **LRUEvictor**
+## Using Tiered Storage
```

Comment:
Maybe "When to enable tiered storage"? up to you

Reply:
I prefer the current heading.